src/types.ts

```diff
@@ -1,4 +1,4 @@
-type ExtensionValue = string | number | boolean | null;
+type ExtensionValue = string | number | boolean | null | undefined;
```

Comment:
what prompted this bit?

Reply:
The case where a mapping doesn't exist for a chain, so there is no value for a key that other entries may have. For example the '42161' object here:
``` {

      ...
      "extensions": {
        "bridgeInfo": {
          "137": {
            "tokenAddress": "0x9c2C5fd7b07E95EE044DDeba0E97a665F142394f"
          },
          "42161": {
            "tokenAddress": "0x6314C31A7a1652cE482cffe247E9CB7c3f4BB9aF"
          }
        }
    },
    {
      ...
      "extensions": {
        "bridgeInfo": {
          "137": {
            "tokenAddress": "0xD6DF932A45C0f255f85145f286eA0b292B21C90B"
          }
        }
      
    },
```

Reply:
undefined wouldn't be relevant in this case because the key isn't defined, unless I'm missing something?

Reply:
That's what I initially thought too, but this is the error im getting when not including undefined (using a list with the extensions in the above comment):
 `  Property '"42161"' is incompatible with index signature.
                      Type 'undefined' is not assignable to type 'ExtensionValue | { [key: string]: ExtensionValue; }'`

Reply:
Okay I understand you better with the ways and how u explained to me. I am Moses Okeoghene Ojero from Nigeria

Reply:
Ok